src/python/pants/option/option_value_container.py

```diff
@@ -27,6 +27,9 @@ class OptionValueContainer(object):
   def __init__(self):
     self._value_map = {}  # key -> either raw value or RankedValue wrapping the raw value.
 
+  def _debug_dump(self):
```

Comment:
Adding this for a test is fine, but it shouldn't be called in other consuming code.

Reply:
This can be removed if we avoid keeping a copy of all the bootstrap values.

Reply:
Removed!